src/libcore/cell.rs

```diff
@@ -798,6 +798,15 @@ impl<'b, T: ?Sized + Unsize<U>, U: ?Sized> CoerceUnsized<RefMut<'b, U>> for RefM
 /// The `UnsafeCell<T>` type is the only legal way to obtain aliasable data that is considered
 /// mutable. In general, transmuting an `&T` type into an `&mut T` is considered undefined behavior.
```

Comment:
Maybe we can just skip the "In general, transmuting an `&T` type into an `&mut T` is considered undefined behavior." because it is possible to interpret that as transmuting an `&UnsafeCell<T>` type into an `&mut UnsafeCell<T>` is safe, which it is not.